cmd/genautocomplete/genautocomplete_test.go

```diff
@@ -21,6 +21,23 @@ func TestCompletionBash(t *testing.T) {
 	assert.NotEmpty(t, string(bs))
 }
 
+func TestCompletionBashStdout(t *testing.T) {
+	originalStdout := os.Stdout
+	tempFile, err := ioutil.TempFile("", "completion_zsh")
+	assert.NoError(t, err)
+	defer func() { _ = tempFile.Close() }()
```

Comment:
These actually get run in the reverse order so the Remove will run before the Close...

This will work fine on Linux but will cause the file to fail to be removed on Windows.

I'd probably write this for the avoidance of doubt!

```go
defer func() {
    _ = tempFile.Close()
    _ = os.Remove(tempFile.Name())
}()
```

Reply:
OK, makes sense. I corrected the other test methods as well.